GAME/TYPES.H

```diff
@@ -175,7 +175,7 @@ struct PHD_VECTOR
 	long y;
 	long z;
 };
-
+#pragma pack(push, 4)
```

Comment:
These pragmas are not supported by the PSX compiler and should be wrapped in #if PC_VERSION

Reply:
Done